File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int length =nums.length;
       
        int nonzero= 0;
        for(int i=0;i<length;i++){
            if(nums[i]!=0){
                nums[nonzero++]=nums[i];
             
            }
        }
        for(int i=nonzero;i<length;i++){
            nums[i]=0;
        }
        
    }
}

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        arr = [0]*len(nums)
        i=0
        for num in nums:
            if num!=0:
                arr[i]=num
                i=i+1
        nums[:]=arr
        